packages/sdk-core/src/model/http/SubmissionUrlInformation.ts

```diff
@@ -80,7 +89,7 @@ export class SubmissionUrlInformation {
             // submit.backtrace.io/universe/token/format
             // by spliting the submission URL by `/` and dropping the last
             // part of the URL, the last element on the list is the token.
-            return submissionUrlParts[submissionUrlParts.length - 2];
+            return submissionUrlParts[submissionUrlParts.length - 2] ?? null;
         }
 
         const url = new URL(submissionUrl);
```

Comment:
You still use `URL` here.

Reply:
As I said - we're using only supported URL API. If something is unsupported I'm creating the URL manually.